### ISSUE TITLE
Automation clips

### ISSUE THREAD

**Issue by AldieNightStar** [gridsound/daw#51]
Would be cool to implement automations like for volume or paning.
Also for some `Custom-written` js plugins for this DAW.

**Comment by mr21** [comment]
Yes indeed, it will be on any paramater :) with the [gsuiDotline](https://github.com/gridsound/gs-ui-components/tree/master/gsuiDotline) component already done, this could be done relatively fast.

**Comment by AldieNightStar** [comment]
And what's about custom js plugins, that i mensioned? IS there some API to do that? :)
I very interested in your DAW. It could become something big, and free, to beat all those expensive DAW :)

**Comment by mr21** [comment]
The DAW itself will stay free yes :)

There are two type of JS plugins: the synthesizers and the effects. About the effects ones we already have an echo by using DelayNodes. So when i will finish to build the effect list related to the mixer and implementing our first effect Echo, then we could see what is the exact pattern we need.
